src/commands/autocomplete/create.ts

```diff
@@ -100,7 +100,7 @@ compinit;\n`
           if (c.hidden) return
           cmds.push({
             id: c.id,
-            description: c.description || '',
+            description: (c.description || '').replace(/`/g, '\''),
```

Comment:
How about escape the backtick rather then replace it

Reply:
Maybe best to move to this to a private function for use here and L127.

Reply:
I tried escaping before replacing, however blackslash backtick does not work either, any suggestion on the escaping?

Reply:
My bad, indeed, I tried to escape with only one backslash. However it should be done with 3 backslashes. That is the reason why I fell back to replacing it.

Reply:
Changes requested have been applied.